.pre-commit-config.yaml

```diff
@@ -12,7 +12,7 @@ repos:
     -   id: rust-style
         name: Rust style
         description: Run cargo fmt on files included in the commit. rustfmt should be installed before-hand.
-        entry: cargo fmt --all -- --check
+        entry: cargo +nightly fmt --all -- --check
```

Comment:
might want to change the Jenkins file to use +nightly as well?

Reply:
the problem is that `cargo` from nix (which is what CI uses) doesn't have `+nightly` option for some reason. Probably cause it's nightly by default. :shrug: `rustfmt` is backwords-compatible though so I'm not sure if we need to (somehow) setup different pre-commit config for the `default.nix` and `ci.nix`. Let's see what the CI says:

bors try

Reply:
Oh that's interesting, wonder why it works like that - but yeah good point, it's already nightly :+1: